### PR TITLE
More simplifications and fixes for JUnit

### PR DESCRIPTION
**Pull request by gzm0**

### CODE REVIEW

File: junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala [hunk 1]
```diff
@@ -124,49 +93,58 @@ final class JUnitExecuteTest(task: JUnitTask, runSettings: RunSettings,
...
     logTestInfo(_.debug, methodName,
         s"finished, took ${getTimeInSeconds()} sec")
 
     // Scala.js-specific: timeouts are warnings only, after the fact
-    val timeInSeconds = getTimeInSeconds()
     val timeout = test.annotation.timeout
     if (timeout != 0 && timeout <= timeInSeconds) {
       richLogger.warn("Timeout: took " + timeInSeconds + " sec, expected " +
-          (timeout.toDouble / 1000) + " sec")
+        (timeout.toDouble / 1000) + " sec")
```
Comment (sjrd) [inline junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala]: The indentation should have been changed here.

File: junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala [hunk 2]
```diff
@@ -149,38 +134,50 @@ final class JUnitExecuteTest(task: JUnitTask, runSettings: RunSettings,
...
       }
     }
 
+    def handleExpected(expectedException: Class[_ <: Throwable])(body: => Unit) = {
+      val wantException = expectedException != classOf[org.junit.Test.None]
+      try {
+        body
+
+        if (wantException)
+          throw new AssertionError("Expected exception: " + expectedException)
```
[the remark this answers is not in the document]
Comment (gzm0) [reply junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala]: Indeed... I wrote a test and found that the error reporting for this was also wrong. Fixed it.

File: junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala (hunk 1 above)
Comment (gzm0) [reply junit-runtime/src/main/scala/org/scalajs/junit/JUnitExecuteTest.scala]: Done.

**Comment by gzm0** [comment]
Updated.

**Comment by gzm0** [comment]
Updated line length in `ExpectTest`.

**Pull request merged**